llvm/lib/Analysis/CostModel.cpp

```diff
@@ -42,25 +44,31 @@ static cl::opt<bool> TypeBasedIntrinsicCost("type-based-intrinsic-cost",
     cl::desc("Calculate intrinsics cost based only on argument types"),
     cl::init(false));
 
+static cl::opt<bool> LibCallBasedIntrinsicCost(
+    "libcall-based-intrinsic-cost",
+    cl::desc("Calculate intrinsics cost using target library info"),
+    cl::init(false));
```

Comment:
Don't understand why this would be an option Remove this?

Reply:
Explained below https://github.com/llvm/llvm-project/pull/123210#discussion_r1965022581